README.md

```diff
@@ -1,76 +1,85 @@
-# up-client-vsomeip-cpp
-## 1. Overview
-This library provides a C++ vsomeip based implementation of uTransport
+# uProtocol C++ VSOMEIP Transport (up-transport-zenoh-cpp)
+
+## Welcome!
+
+This library provides a vsomeip-based uProtocol transport for C++ uEntities.
 
 *_IMPORTANT NOTE:_ This project is under active development*
 
-This module contains the implementation for pub-sub and RPC API`s defined in the uProtocol spec
+This module contains the vsomeip implementation of the Layer 1 `UTransport` API
+from [up-cpp][cpp-api-repo].
+
+## Getting Started
 
-## 2. Getting Started
-### 2.1. Requirements:
+### Requirements:
 - Compiler: GCC/G++ 11 or Clang 13
-- Ubuntu  : 22.04
-- conan   : 1.59 or latest 2.X
-- cmake   : 3.20.0+
-- ninja   : 1.10.0+ (optional, can instead use 'make' as well)
+- Conan : 1.59 or latest 2.X
 
-### 2.2. Dependencies
-#### 2.2.1. up-cpp
-Install up-cpp library (version v0.1.2-dev): https://github.com/eclipse-uprotocol/up-cpp
+#### Conan packages
 
-#### 2.2.1. vsomeip
-- Install boost (version: 1.84.0): https://www.boost.org/users/history/version_1_84_0.html
-```
-$ cd boost
-$ ./bootstrap.sh --prefix=<installation-directory> # can be left as default /usr/local
-$ ./b2
-$ sudo ./b2 install
-```
+Using the recipes found in [up-conan-recipes][conan-recipe-repo], build these
+Conan packages:
 
-- Install vsomeip (version: 3.4.10):
+1. [up-core-api][spec-repo] - `conan create --version 1.5.8 --build=missing up-core-api/developer`
+2. [up-cpp][cpp-api-repo] - `conan create --version 0.2.0 --build=missing up-cpp/developer`
```

Comment:
Surely there is a vsomeip requirement that must be installed somehow. Could that be a conan recipe? Or is there a repo with build instructions?

Reply:
I need to check if its possible with conan so far old readme refers to 

`$ git clone https://github.com/COVESA/vsomeip.git
$ git cd vsomeip
$ git checkout 3.4.10
$ mkdir build
$ cd build
$ cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr/local/ -G Ninja ..
$ ninja
$ sudo ninja install`

Reply:
vsomeip doesn't have a conan recipe. We can write one for them. I raised an issue with covesa/vsomeip and they seem to be fine with it. But until we have that, we have to do this manually